src/controllers/User.controller.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+'use strict';
+
+const UserService = require('../services/User.Service');
+
+const get = async(request, response) => {
+  try {
+    const allUsers = await UserService.getAll();
+
+    if (!allUsers) {
+      response.status(404).send('Not Found: The specified entity does not exist');
+    }
+
+    response.statusCode = 200;
+    response.send(allUsers);
+  } catch (error) {
+    console.log(error);
```

Comment:
Instead of just logging the error, provide more context or information to the client about what went wrong.